app/components/tabSelector/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { Text, StyleSheet, View, Button } from "react-native";
+

```

Comment:
you can declare and export a type for tab selector item, like so 
```suggestion

export type TabSelectorItemType = { 
   title: string, 
   component: React.ReactNode // (I think this is the correct type for components
}
```